match.py

```diff
@@ -105,17 +116,34 @@ def match10_no_strict(flow_dict, args):
     return flow_dict
 
 
-def match13_no_strict(flow_to_install, stored_flow_dict):
-    """Match a flow that is either exact or more specific (non-strict) (OF1.3).
-
-    Return the flow if any fields match, otherwise, return False.
-    """
+def _match_cookie(flow_to_install, stored_flow_dict):
+    """Check if a the cookie and its mask matches between the flows."""
     cookie = flow_to_install.get("cookie", 0) & flow_to_install.get("cookie_mask", 0)
     cookie_stored = stored_flow_dict.get("cookie", 0) & flow_to_install.get(
         "cookie_mask", 0
     )
     if cookie and cookie != cookie_stored:
         return False
+    return True
+
+
+def _match_keys(flow_to_install, stored_flow_dict, flow_to_install_keys):
```

Comment:
This was extracted to be re-used, but to add support for masked values we'll cover on this issue #24 in the future.